REFERENCE.md

```diff
@@ -112,11 +112,27 @@ Data type: `Stdlib::Absolutepath`
 
 Path to the directory where the module stores ferm configuration files
 
-##### `disable_conntrack`
+##### `forward_disable_conntrack`
+
+Data type: `Boolean`
+
+Disable the generation of conntrack rules for the FORWARD chain
+
+Default value: `true`
```

Comment:
Now the module is with `puppet-string`  and generated `REFERENCE.md`, so i prefer to not write manually default values. This avoid to have some doc not synced with what is in the code, since `REFERENCE.md` handle this.